packages/rainbowkit/src/components/ConnectOptions/ConnectOptions.tsx

```diff
@@ -1,10 +1,16 @@
-import React from 'react';
+import React, { useContext } from 'react';
 import { isMobile } from '../../utils/isMobile';
+import { RainbowButtonContext } from '../RainbowKitProvider/RainbowButtonContext';
 import { DesktopOptions } from './DesktopOptions';
 import { MobileOptions } from './MobileOptions';
 
 export default function ConnectOptions({ onClose }: { onClose: () => void }) {
-  return isMobile() ? (
+  const { connector } = useContext(RainbowButtonContext);
+
+  // If you do inspect element on mobile then switch to
+  // desktop view the rainbowkit connect modal will pop up.
+  // In this case we don't want that if we are using a custom connector
+  return isMobile() || connector ? (
     <MobileOptions onClose={onClose} />
   ) : (
     <DesktopOptions onClose={onClose} />
```

Comment:
We would do a switch here to `MobileLinking`

Reply:
Aha makes sense. Can you give more context on that ? I believe we can create a component called `MobileLinking` that does this type of logic ? Is that the case ?